app/javascript/components/rooms/RoomsList.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Row, Col, Container, Button, Spinner, Form, FormControl } from "react-bootstrap";
+import RoomCard from "./RoomCard";
+import useRooms from "../../hooks/queries/rooms/useRooms";
+import ButtonLink from "../stylings/buttons/ButtonLink";
+
+export default function RoomsList() {
+
+  const { isLoading, data: rooms } = useRooms();
+  if (isLoading) return <Spinner />
+
+  return (
+    <>
+      <Container>
```

Comment:
Is Container necessary for Bootstrap grid?
It creates the unwanted gap (please see redline below).
I (usually) use Container only twice, once for Header, and once for the rest of the body.

Container can be removed. 
Container-fluid could be used (it removes the gap) if needed.
Or a simple <div> if you need to wrap things up.

![image](https://user-images.githubusercontent.com/43917914/161778509-d743bcc6-7b17-4de6-8957-2dfbb9bb5704.png)

Reply:
The container helps align everything together, and personally I find the spacing on the side nice. This should be a team vote though @farhatahmad @KH-Amir-TN